### ISSUE TITLE
restafval - status: unknown/ in calendar it shows with other name

### ISSUE THREAD

**Issue by luckyzor** [pippyn/Home-Assistant-Sensor-Afvalbeheer#358]
Hello, 

i have some issues with this integration, don't know how to solve them:

![Screenshot_20221107_121404](https://user-images.githubusercontent.com/31155573/200297460-a781b631-c019-4480-abfc-8902a21d12fa.jpg)

As you can see in this SS, the sensor for the gft, it have 2 values: gft,gft on the calendar too. Don't know why.

For the restafval the status is unknown but in the calendar the "déchets résiduelles" is there in the good date.

Can this be fixed?

Thanks for you work

**Comment by pippyn** [comment]
What is the config of the sensor?

**Comment by luckyzor** [comment]
![Screenshot_20221107_122524~2](https://user-images.githubusercontent.com/31155573/200299366-bb540414-c254-4a78-8b0e-08c602932548.jpg)

Its a working adresse, same address as I use in the app.

**Comment by pippyn** [comment]
please add your config as code, not a photo? And also provide a working address, so I can test it myself.

**Comment by luckyzor** [comment]
> please add your config as code, not a photo. And also provide a working address, so I can test it myself.


```
afvalbeheer:
  wastecollector: RecycleApp
  resources:
    - restafval
    - gft
    - papier
    - pmd
  postcode: 7030
  streetnumber: 9
  streetname: Clos Theodore Wright
  cityname: Saint-Symphorien
  upcomingsensor: 1 # (optional)
  dateformat: "%d-%m-%Y" # (optional)
  builtinicons: 1 # (optional)
  builtiniconsnew: 1
  dutch: 0 # (optional)
```

**Comment by pippyn** [comment]
The issue is caused by RecycleApp. It reports 2 GFT collections on the same day:
![image](https://user-images.githubusercontent.com/7591990/200502971-4e1398b7-fa9b-4ff9-93d1-e92523eef1d9.png)

**Comment by luckyzor** [comment]
And concerning the unknown status of the restafval?
Did you found something?

**Comment by pippyn** [comment]
I just released v5.0.10 which fixes that issue
